ompi/win/win.c

```diff
@@ -167,18 +167,18 @@ config_window(void *base, size_t size, int disp_unit,
                                      MPI_WIN_SIZE, size, true);
```

Comment:
@hjelmn Actually, `size_t` is unsigned here, and `MPI_Aint` is a `ptrdiff_t`, which is signed.  Can't this cause a problem in some cases?